lib/v0_bindings_extensions.rb

```diff
@@ -6,11 +6,35 @@
 
   # If we're running the task to generate model bindings, there's nothing yet
   # to monkey patch, so bail.
-  next if RakeUtils.running_task?(/generate_model_bindings/)
+  next if RakeUtils.running_task?(/generate_model_bindings|generate_swagger/)
+
+  Api::V0::Bindings::Events.class_exec do
+    alias_method :original_valid?, :valid?
+    def valid?
+      original_valid? && events.all?(&:valid?)
+    end
+
+    alias_method :original_list_invalid_properties, :list_invalid_properties
+    def list_invalid_properties
+      event_invalid_properties = events.map.with_index do |event,ii|
```

Comment:
suggestion..to not use cryptic abbreviations like i, x, y, ii, etc.  and to say what it really is.  index.

Reply:
fixed.  old habits :-)  (from when I was annoyed with people using just a single letter `i` :-)